### ISSUE TITLE
Web Store Extension

### ISSUE THREAD

**Issue by linuxtopia** [Eloston/ungoogled-chromium#988]
Hello,

Many of us can not drag&drop Web Store or other CRX files to extension on UGC. Then please bundle UGC releases with Web Store extension then no need to drag&drop other crx files by chrome store.

Thanks

**Comment by PF4Public** [comment]
But what about those, who don't need them?

**Comment by linuxtopia** [comment]
> But what about those, who don't need them?

Is it hard to remove by clicking?! It is "impossible"  to add those who need them though...

**Comment by Magus-Sama** [comment]
You don't need to drag and drop - go to chrome://flags and change "Handling of extension MIME type requests" to Always Prompt to Install. Then, just go to your FM and open the crx file with Ungoogled Chromium.
That being said, I do second having the Chromium Web Store extension installed by default for sheer convenience - it makes it much easier to install addons. Whoever doesn't need it can simply remove it. What's the harm?

**Comment by seanking2919** [comment]
This is a common issue that's been going for a while. Many of us are trying to figure out how to fix this: https://github.com/Eloston/ungoogled-chromium/issues/923

**Comment by linuxtopia** [comment]
> You don't need to drag and drop - go to chrome://flags and change "Handling of extension MIME type requests" to Always Prompt to Install. Then, just go to your FM and open the crx file with Ungoogled Chromium.
> That being said, I do second having the Chromium Web Store extension installed by default for sheer convenience - it makes it much easier to install addons. Whoever doesn't need it can simply remove it. What's the harm?

unfortunately, Windows right click and open with menu can not accept UGC as a program then can not open crx file with it. so weird but the case is that.

**Comment by xnaas** [comment]
I can't get any CRX files to install at all on:

```
Chromium	80.0.3987.149 (Developer Build) (64-bit)
Revision	5f4eb224680e5d7dca88504586e9fd951840cac6-refs/branch-heads/3987_137@{#16}
OS	Windows 10 OS Version 1909 (Build 18363.752)
JavaScript	V8 8.0.426.27
Flash	(Disabled)
User Agent	Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.149 Safari/537.36
Command Line	"C:\Users\xnaas\AppData\Local\Chromium\Application\chrome.exe" --flag-switches-begin --extension-mime-request-handling=always-prompt-for-install --flag-switches-end --enable-audio-service-sandbox --file-url-path-alias="/gen=C:\Users\xnaas\AppData\Local\Chromium\Application\gen"
Executable Path	C:\Users\xnaas\AppData\Local\Chromium\Application\chrome.exe
Profile Path	C:\Users\xnaas\AppData\Local\Chromium\User Data\Default
Variations	5e3a236d-59e286d0
```

**Comment by Eloston** [comment]
There's no easy way to bundle Chromium Web Store because each platform is pretty different, and most of them don't have an easy way to bundle removable extensions I believe.

Also, this wouldn't really be a problem if we fixed #923 and https://github.com/ungoogled-software/ungoogled-chromium-windows/issues/23. I think those are more important anyway, so closing as wontfix.
